package.json

```diff
@@ -12,8 +12,9 @@
     "clean": "rm -rf build/ .parcel-cache",
     "start": "parcel --dist-dir build-dev --no-cache",
     "checkTs": "tsc --noEmit",
-    "lint": "prettier src internals --check",
-    "lint:fix": "prettier src internals --write",
+    "lint": "eslint --ext js,ts,tsx ./",
```

Comment:
CI says:
```
$ eslint --ext js,ts,tsx ./

/home/runner/work/explorer/explorer/src/app/pages/HomePage/ParaTimeSelector/HelpScreen/index.tsx
Warning:   [7](https://github.com/oasisprotocol/explorer/actions/runs/4194095200/jobs/7271812264#step:5:8)7:1[9](https://github.com/oasisprotocol/explorer/actions/runs/4194095200/jobs/7271812264#step:5:10)  warning  The ref value 'swiperElRef.current' will likely have changed by the time this effect cleanup function runs. If this ref points to a node rendered by React, copy 'swiperElRef.current' to a variable inside the effect, and use that variable in the cleanup function  react-hooks/exhaustive-deps

✖ 1 problem (0 errors, 1 warning)
```

I think we should throw on warnings too, like wallet
https://github.com/oasisprotocol/oasis-wallet-web/blob/c3da68aba502ba7d26f3067a6ef544b477a282d3/.github/workflows/build-test.yaml#L36

Reply:
OK thanks, I cherry-picked the fix.

Reply:
After the latest commit, the warning is gone.